docs/source/user_guides/superset_guide.rst

```diff
@@ -0,0 +1,109 @@
+Superset guide
+---------------------
+
+Description
+~~~~~~~~~~~
+
+| The Dialog Flow Stats module can be used to obtain and visualize usage statistics for your service.
+| Data aggregation relies on the `OpenTelemetry protocol <#>`_ and the `OpenTelemetry collector <#>`_ along with `Clickhouse <https://clickhouse.com/>`_ as an OLAP storage.
+| Interactive visualization is powered by `Apache Superset <https://superset.apache.org/>`_.
+| All the mentioned services are shipped as Docker containers, including a pre-built Superset image that ensures API compatibility.
+
+Collection procedure
+~~~~~~~~~~~~~~~~~~~~
+
+**Installation**
+
+.. code-block:: shell
+
+    pip install dff[stats]
+
+**Launching services**
+
+.. code-block:: shell
+    :linenos:
+
+    # clone the original repository to access the docker-compose file
+    git clone https://github.com/deeppavlov/dialog_flow_framework.git
+    # launch the required services
+    docker-compose up otelcol clickhouse dashboard
+
+**Collecting data**
+
+Collecting data is done by means of instrumenting your conversational service before you run it.
+DFF tutorials showcase all the necessary steps, needed to achieve that. We will run
+`one of those files <../tutorials/tutorials.stats.1_extractor_functions.py>`_ 
+in order to obtain sample data points to visualize.
+
+.. code-block:: shell
+
+    export DISABLE_INTERACTIVE_MODE=1 && python tutorials/stats/1_extractor_functions.py
+
+Displaying the data
+~~~~~~~~~~~~~~~~~~~
+
+In order to display the Superset dashboard, you should update the default configuration with the credentials of your database.
+The configuration can be optionally saved as a zip archive for inspection / debug.
+
+You can set most of the configuration options using a YAML file.
+
+.. code-block:: yaml
+    :linenos:
+
+    # config.yaml
+    db:
+        type: clickhousedb+connect
+        name: test
+        user: username
+        host: clickhouse
+        port: 8123
+        table: otel_logs
+
+The file can then be used to parametrize the configuration script.
+Password values will be prompted once you run the following command:
+
+.. code-block:: shell
+
+    dff.stats config.yaml -P -dP -U superset --outfile=config_artifact.zip
```

Comment:
I guess this command should be also able to pick up passwords from env variables, don't you think?

Reply:
Since it's meant to be agnostic to database type, it's not clear which environment variable it should use for that. We know the variable name for Superset, but it would be strange to pick only one password from the environment

Reply:
Fair enough; do we plan any other databases, not superet?

Reply:
Not at the moment, but I still see no reason to hardcode it. For instance, users can pass the environment variables to the cli interface directly if they need to.